docs/src/languages/php/extensions.md

```diff
@@ -60,110 +61,111 @@ You can disable those by adding them to the `disabled_extensions` list.
+| amqp             |     |     |     | *   | *   | *   | *   | *   |     |
+| apc              | *   | *   |     |     |     |     |     |     |     |
+| apcu             | *   |     | *   | *   | *   | *   | *   | *   | *   |
+| apcu_bc          |     |     |     | *   | *   | *   | *   | *   |     |
+| applepay         |     |     |     | *   | *   |     |     | *   |     |
+| bcmath           |     |     |     | *   | *   | *   | *   | *   | *   |
+| blackfire        | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| bz2              |     |     |     | *   | *   | *   | *   | *   | *   |
+| calendar         |     |     |     | *   | *   | *   | *   | *   | *   |
+| ctype            |     |     |     | *   | *   | *   | *   | *   | *   |
+| curl             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| dba              |     |     |     | *   | *   | *   | *   | *   | *   |
+| dom              |     |     |     | *   | *   | *   | *   | *   | *   |
+| enchant          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| event            |     |     |     |     | *   | *   | *   | *   |     |
+| exif             |     |     |     | *   | *   | *   | *   | *   | *   |
+| ffi              |     |     |     |     |     |     |     | *   | *   |
+| fileinfo         |     |     |     | *   | *   | *   | *   | *   | *   |
+| ftp              |     |     |     | *   | *   | *   | *   | *   | *   |
+| gd               | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| gearman          | *   | *   | *   |     |     |     |     |     |     |
+| geoip            | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| gettext          |     |     |     | *   | *   | *   | *   | *   | *   |
+| gmp              | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| http             | *   | *   |     |     |     |     |     | *   |     |
+| iconv            |     |     |     | *   | *   | *   | *   | *   | *   |
+| igbinary         |     |     |     | *   | *   | *   | *   | *   | *   |
+| imagick          | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| imap             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| interbase        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| intl             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| ioncube          |     |     |     | *   | *   | *   |     |     |     |
+| json             |     |     | *   | *   | *   | *   | *   | *   | *   |
+| ldap             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| mailparse        |     |     |     | *   | *   | *   |     | *   |     |
+| mbstring         |     |     |     | *   | *   | *   | *   | *   | *   |
+| mcrypt           | *   | *   | *   | *   | *   |     |     |     |     |
+| memcache         | *   | *   | *   |     |     |     |     |     |     |
+| memcached        | *   | *   | *   | *   | *   | *   |     | *   |     |
```

Comment:
This isn't a 3rd party at this point, is it?

Reply:
Yes its third party but I've completely missed it ... its in progress thanks

Reply:
I've updated the image and the doc will be included on the next php8 image release